.travis.yml

```diff
@@ -1,22 +1,32 @@
-os:
-  - linux
-  - osx
-env:
-  - TEST_SUITE=testinstall
-  - TEST_SUITE=testtravis
-  - TEST_SUITE=testbugfix
 language: c
-compiler:
-  - gcc
-  - clang
-branches:
-  only:
-    - master
-    - hpcgap-default
+matrix:
+  include:
+    - os: linux
+      env: TEST_SUITE=testtravis
+      compiler: gcc
+    - os: osx
+      env: TEST_SUITE=testtravis
+      compiler: clang
+    - os: linux
+      env: TEST_SUITE=testinstall
+      compiler: gcc
+    - os: linux
+      env: TEST_SUITE=testbugfix
+      compiler: gcc
+    - os: linux
+      env: TEST_SUITE=makemanuals
```

Comment:
For makemanuals to work, one should check first `if [ $TEST_SUITE = 'makemanuals' ]` - see the master branch how it is organised. It may still not work, then I suggest to exclude this from `.travis.ym` for now and create an issue in this case. Actually, in HPC-GAP branch `make manuals` crashes, but `../../bin/gap.sh makedocrel.g` works.